project/Merging.scala

```diff
@@ -2,23 +2,16 @@ import sbtassembly.{MergeStrategy, PathList}
 
 object Merging {
   def customMergeStrategy(oldStrategy: (String) => MergeStrategy):(String => MergeStrategy) = {
-    case x if x.endsWith("io.netty.versions.properties") => MergeStrategy.discard
```

Comment:
Edmund ran sbt assembly without these lines and there were no errors.

Reply:
What are these `MergeStrategy`s? Are we removing them because we think they're unnecessary or because they're incompatible?